src/fa/game_updater/worker.py

```
@@ -232,21 +235,21 @@ def _download(self, target_path: str, url: str, params: dict[str, str]) -> None:
 
         for attempt in range(10):  # after download antimalware can interfere in our update process
-            if self.fa_patcher.patch(exe_path, version):
+            if self.fa_patcher.patch(exe_path, self.version):
                 return
             logger.warning("Could not open fa exe for patching. Attempt #%d", attempt + 1)
             thread = self.thread()
             assert thread is not None
             thread.msleep(500)
         else:
-            raise UpdaterFailure("Could not update FA exe to the correct version")
+            raise UpdaterFailure("Could not update FA exe to version '%d'", self.version)
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)